ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandStore.java

```diff
@@ -183,6 +183,10 @@ public void ensureConsumedPast(final long seqNum, final Duration timeout)
     }
   }
 
+  public boolean empty() {
+    return commandTopic.getCommandTopicConsumerPosition() == 0;
```

Comment:
I feel like this is a little misleading. Just because we haven't started processing the stream doesn't mean it's empty. We can use `KafkaConsumer#endOffsets` to determine if a topic is empty (it will return 0)

Reply:
yeah good call - this went through a couple rounds of refactoring/renaming. Using KafkaConsumer#endOffsets definitely makes more sense the way it's structured now.